content/en/blog/_posts/2023-07-18-swap-beta1-graduation/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: blog
+title: "Kubernetes 1.28: NodeSwap graduates to Beta"
+date: 2023-07-18
+slug: swap-beta1-1.28-2023
+---
+
+**Author:** Itamar Holder (Red Hat)
+
+The 1.22 release introduced Alpha support for configuring swap memory usage for
```

Comment:
I think we may add a link to the alpha blog in https://kubernetes.io/blog/2021/08/09/run-nodes-with-swap-alpha/ here.

Reply:
Great idea!
Added, thanks!